clearwater-queue-manager.root/usr/share/clearwater/clearwater-queue-manager/scripts/modify_nodes_in_queue.py

```diff
@@ -58,28 +58,42 @@ def make_key(site, clearwater_key, queue_key):
 queue_syncer = EtcdSynchronizer(NullPlugin(queue_key), local_ip, site, clearwater_key, node_type)
 
 if operation == "add":
-    logging.info("Adding %s to queue to restart" % (local_ip + "-" + node_type))
+    logging.debug("Adding %s to queue to restart" % (local_ip + "-" + node_type))
```

Comment:
Are none of these logs useful to a customer at the info level?
I am assuming these are the logs you were calling spammy, but would a combination of the above and below logs not be useful in production? i.e. "Node <ip><name> successfully added to/removed from restart queue"